intel_extension_for_transformers/llm/library/jblas/jblas/jit_blas_utils.h

```diff
@@ -29,6 +29,7 @@
 #define CompileAVX512F() (defined(__GNUC__) && (__GNUC__ >= 6))
 #define CompileAVX2() (defined(__GNUC__) && (__GNUC__ >= 5))
 #define CompileAMX() (defined(__GNUC__) && (__GNUC__ >= 11))
+#define CompileBF16Kernel() (defined(__GNUC__) && (__GNUC__ >= 10))
```

Comment:
what's the differences between ```CompileBF16Kernel``` and ```CompileBF16```

Reply:
I'm not sure. See https://github.com/intel-sandbox/jblas/pull/23/files#diff-4f85ef22dde7dcf1f27a8db354d0c00740b2143e744a33d54aafe3fd874e1307R32.
(internal access required).